examples/aggregator/module/rust/src/lib.rs

```diff
@@ -157,9 +157,26 @@ impl Aggregator for AggregatorNode {
     }
 }
 
-oak::entrypoint!(oak_main => |_in_channel| {
+oak::entrypoint!(grpc_worker => |in_channel| {
     oak::logger::init_default();
     let dispatcher = AggregatorDispatcher::new(AggregatorNode::new());
-    let grpc_channel = oak::grpc::server::init("[::]:8080").expect("could not create gRPC server pseudo-Node");
-    oak::run_event_loop(dispatcher, grpc_channel);
+    oak::run_event_loop(dispatcher, oak::io::Receiver::<grpc::Invocation>::new(in_channel));
+});
+
+oak::entrypoint!(oak_main => |_in_channel| {
+    oak::logger::init_default();
+    let grpc_channel =
+        oak::grpc::server::init("[::]:8080").expect("could not create gRPC server pseudo-Node");
+    let (invocation_sender, invocation_receiver) =
+        oak::io::channel_create::<oak::grpc::Invocation>()
+            .expect("could not create gRPC invocation channel");
+    oak::node_create(
```

Comment:
Does it mean that the initial Node and the worker Node are defined in the same Wasm file?
Is it supposed to be like this, or it's better to have Node=module?

Reply:
In this case there is no use case in splitting them, so it is much more convenient to do it in the same module. But of course, if we plan to have a reusable aggregator module, we should consider splitting them in the future.